MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -17,15 +17,18 @@ final class FeedViewModel {
 	var title: String {
 		Localized.Feed.title
 	}
-	
+
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
-	
+	var onError: Observer<String>?
```

`onError: Observer<String>?` could be `onErrorStateChange: Observer<String?>?`, so the ViewModel decides when to show/hide errors (instead of the view controller). It's often easier to manage state when all changes are centralized.

For example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed.